src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,50 @@
+@import "../utils/variables.scss";
+
+.calendar {
+  width: 727px;
+  display: flex;
+  gap: 1px;
+  flex-wrap: wrap;
+  padding: 10px;
+
+  &--start-day {
+    $start-days:
+      "mon" 0 * $day-width,
+      "tue" 1 * $day-width,
+      "wed" 2 * $day-width,
+      "thu" 3 * $day-width,
+      "fri" 4 * $day-width,
+      "sat" 5 * $day-width,
+      "sun" 6 * $day-width;
+
+    @each $day, $start-day in $start-days {
+      &--#{$day} {
+        margin-left: $start-day;
+      }
+    }
+  }
+
+  &__day {
+    background-color: grey;
+    box-sizing: border-box;
+    height: $day-height;
+    width: 100px;
+    text-align: center;
+    font-size: 30px;
+    line-height: $day-height;
+    border: solid 1px black;
+
+    &:hover {
+      background-color: pink;
+      cursor: pointer;
+      position: relative;
+      bottom: 10px;
```

Comment:
The hovered cell should move up by 20px, but currently, it moves up by 10px. Change the bottom value to 20px: bottom: 20px;